### PR TITLE
Fix SoundManager handling when no sound devices exist

### PR DESCRIPTION
**Pull request by kstvr32**
Supersedes GTNewHorizons/Hodgepodge#420 as a fix for the SoundManager NPE crashes

Fixes https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/17190
Fixes https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/17190
Fixes https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/16281
Fixes https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/12848
Fixes https://github.com/GTNewHorizons/Hodgepodge/issues/406

Adds null check that is present in LWJGL2 when initializing OpenAL sound.

### CODE REVIEW

File: src/main/java/org/lwjglx/openal/AL.java [hunk 1]
```diff
@@ -64,6 +64,10 @@ public static void create(String deviceArguments, int contextFrequency, int cont
 
         long deviceHandle = org.lwjgl.openal.ALC10.alcOpenDevice(defaultDevice);
 
+        if (deviceHandle == 0) {
+            throw new org.lwjgl.LWJGLException("Could not open ALC device");
```
Comment (eigenraven) [inline src/main/java/org/lwjglx/openal/AL.java:68]: The mappings should only throw the lwjglx exceptions, not lwjgl - change this here and remove the throws clauses above. The class references are remapped from `x` to no `x` in an asm transformer later

**Review by eigenraven — APPROVED** [review]

File: src/main/java/org/lwjglx/openal/AL.java (hunk 1 above)
Comment (kstvr32) [reply src/main/java/org/lwjglx/openal/AL.java]: Changed to `lwgjlx` variant. I think we still need the throws above because the create calls are chained.

**Comment by Edgars-Cirulis** [comment]
Does that commit resumes the sound?

**Comment by kstvr32** [comment]
This changes back to standard 1.7.10 behavior of turning to silent mode. 

You can manually reload the sound by pressing F3 + T

**Pull request merged**